pydoc/developer/MEMORY.md

```diff
@@ -0,0 +1,16 @@
+.. _memory_usage:
```

Comment:
Again, this label won't work natively in markdonw. Update to 
<pre><code>
```{eval-rst}  
.. _memory_usage:
```
</code></pre>

or remove since they are not used.